test/hotspot/jtreg/runtime/cds/appcds/aotClassLinking/BulkLoaderTest.java

```diff
@@ -30,7 +30,6 @@
  * @test id=static
  * @requires vm.cds.supports.aot.class.linking
  * @comment work around JDK-8345635
```

Comment:
The `* @comment work around JDK-8345635` lines should also be removed from all the CDS tests.